README.md

```diff
@@ -140,13 +130,13 @@ docker-compose --env-file ./.env -f docker/docker-compose.build.yml up -d
 Open http://localhost:3333 in your browser and accomplish these steps:
 
 1. Create a new user via _Get Started_ (this first user will get the role `ADMIN`)
-1. Go to the _Market Data_ tab in the _Admin Control Panel_ and click _Gather All Data_ to fetch historical data
-1. Click _Sign out_ and check out the _Live Demo_
+2. Go to the _Market Data_ tab in the _Admin Control Panel_ and click _Gather All Data_ to fetch historical data
```

Comment:
No, it's not incorrect, they get rendered correctly. It's actually a trick that you don't have to modify every item if you change something in the list 🥲

Reply:
Well I learned something ! :D

Reply:
> Well I learned something ! :D

Great! In my opinion, that's the point of open source: to learn new stuff 🤓